packages/api/test/scripts/car.js

```diff
@@ -0,0 +1,17 @@
+import * as Block from 'multiformats/block'
+import * as Raw from 'multiformats/codecs/raw'
+import { sha256 } from 'multiformats/hashes/sha2'
+import * as CAR from '../../src/utils/car.js'
+
+// @ts-ignore
```

Comment:
nit: I usually like to add the reasons for adding ts-ignore next to it. This helps us cleaning them up in the future. I don't know the exact pattern in this project though

Reply:
I did not realise you were allowed to add a message so I haven't been doing it so far. I agree a message is useful. does it just need:

```suggestion
/**
 * @param {string} str Data to encode into CAR file.
 */
```